Gemfile

```diff
@@ -4,9 +4,8 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 # Specify your gem's dependencies in importmap-rails.gemspec.
 gemspec
 
-rails_version = ENV["RAILS_VERSION"] || "6.1.0"
-gem "rails", "~> #{rails_version}"
-
+gem "appraisal"
+gem "rails", "~> 6.0.0"
```

Comment:
This file defaulted the Rails version to 6.1, but the [gemspec specifies 6.0](https://github.com/rails/jsbundling-rails/blob/44919fe2ecc11fd23b26fc39692615d87656da7f/jsbundling-rails.gemspec#L14).  Which is correct?  (Or should we jump to Rails 7.0?)